TrackletAlgorithm/TrackBuilder.h

```diff
@@ -4,19 +4,32 @@
 #include "TrackletParameterMemory.h"
 #include "FullMatchMemory.h"
 #include "TrackFitMemory.h"
+#include "TrackBuilder_parameters.h"
 
+static const unsigned short kNBitsTBBuffer = 1;
```

Comment:
@aryd Somehow this unused constant got added back in. Please remove it.
```suggestion
```

Reply:
Fixed.